firefox_privacy_notice/fx56_notice.md

```diff
@@ -57,35 +57,31 @@ By default on desktop versions of Firefox, we will ask you to share a report wit
 
 Read the full documentation [here](https://firefox-source-docs.mozilla.org/toolkit/crashreporter/crashreporter/index.html).
 
-### Measure and support our marketing+
+### Measure and support our marketing {: #thirdparty } ** **{: #referraltracking }
 
 * __Campaign and Referral Data__: This helps Mozilla understand the effectiveness of our marketing campaigns.   
 
-  * _On Desktop_: Firefox by default sends Mozilla HTTP data that may be included with Firefox’s installer.  This enables us to determine the website domain or advertising campaign (if any) that referred you to our download page.  
-
-    Read the [documentation](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/environment.html#attribution) or [opt-out](https://support.mozilla.org/kb/desktop-privacy) before installation.
-
-  * _On iOS and Android_: Firefox by default sends mobile campaign data to Adjust, our analytics vendor, which has its own [privacy policy](https://www.adjust.com/privacy_policy/).  Mobile campaign data includes a Google advertising ID, IP address, timestamp, country, language/locale, operating system, and app version.  
+    _On Desktop_: Firefox by default sends Mozilla HTTP data that may be included with Firefox’s installer.  This enables us to determine the website domain or advertising campaign (if any) that referred you to our download page. Read the [documentation](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/environment.html#attribution) or [opt-out](https://support.mozilla.org/kb/desktop-privacy) before installation.
 
-    Read the [documentation](https://firefox-source-docs.mozilla.org/mobile/android/fennec/adjust.html).
+    _On iOS and Android_: Firefox by default sends mobile campaign data to Adjust, our analytics vendor, which has its own [privacy policy](https://www.adjust.com/privacy_policy/).  Mobile campaign data includes a Google advertising ID, IP address, timestamp, country, language/locale, operating system, and app version. Read the [documentation](https://firefox-source-docs.mozilla.org/mobile/android/fennec/adjust.html).
```

Comment:
This one, and the one above "_On Desktop_", are they supposed to make up a single unordered list? If so, then the indentations are not needed.

Reply:
the "on desktop" and "on ios and android" items were supposed to be an unordered list underneath campaign and referral data. due to the markdown limitation you pointed out i have just made them regular text within "campaign and referral data" listitem, but with formatting